rest-tests/src/test/java/io/syndesis/qe/rest/tests/integrations/steps/ConnectionSteps.java

```diff
@@ -0,0 +1,186 @@
+        table.add(Arrays.asList("password", account.getProperty("password")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create FTP connection")
+    public void createFTPConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("ftp").get();
+        table.add(Arrays.asList("connector", "ftp"));
+        table.add(Arrays.asList("host", account.getProperty("host")));
+        table.add(Arrays.asList("port", account.getProperty("port")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create Dropbox connection")
+    public void createDropboxConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("QE Dropbox").get();
+        table.add(Arrays.asList("connector", "dropbox"));
+        table.add(Arrays.asList("accessToken", account.getProperty("accessToken")));
+        table.add(Arrays.asList("clientIdentifier", account.getProperty("clientIdentifier")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create HTTP connection")
+    public void createHTTPConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("http").get();
+        table.add(Arrays.asList("connector", "http"));
+        table.add(Arrays.asList("baseUrl", account.getProperty("baseUrl")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create Kafka connection")
+    public void createKafkaConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("kafka").get();
+        table.add(Arrays.asList("connector", "kafka"));
+        table.add(Arrays.asList("brokers", account.getProperty("brokers")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create SalesForce connection")
+    public void createSalesForceConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("QE Salesforce").get();
+        table.add(Arrays.asList("connector", "salesforce"));
+        table.add(Arrays.asList("cliendId", account.getProperty("cliendId")));
+        table.add(Arrays.asList("clientSecret", account.getProperty("clientSecret")));
+        table.add(Arrays.asList("loginUrl", account.getProperty("loginUrl")));
+        table.add(Arrays.asList("userName", account.getProperty("userName")));
+        table.add(Arrays.asList("password", account.getProperty("password")));
+        createConnection(DataTable.create(table));
+    }
+
+    @Given("^create Twitter connection")
+    public void createTwitterConnection() {
+        final List<List<String>> table = new ArrayList<>();
+        final Account account = accountsDirectory.getAccount("twitter_talky").get();
```

Comment:
What about connection using "twitter_listen" account? Or other accounts?

Reply:
it wasn't used so I didn't include that, but I'll rework as "create Twitter connection with X account"

Reply:
In case it's not really used anywhere, I'll leave the decision whether to include the possibility to specify the account up to you.

Reply:
don't worry, be happy